experimental/examples/falling-balls-mpp/gradle.properties

```diff
@@ -9,3 +9,6 @@ kotlin.mpp.enableGranularSourceSetsMetadata=true
 # Enable kotlin/native experimental memory model
 kotlin.native.binary.memoryModel=experimental
 compose.desktop.verbose=true
+org.jetbrains.compose.experimental.skikojs.enabled=true
```

Comment:
better name it jscanvas.

skiko is implementation details. Canvas is main distinguish feature

Reply:
Done